daemon/snap.go

```diff
@@ -178,8 +179,14 @@ func mapLocal(about aboutSnap) map[string]interface{} {
 		status = "active"
 	}
 
+	appNames := make([]string, 0, len(localSnap.Apps))
+	for appName := range localSnap.Apps {
+		appNames = append(appNames, appName)
+	}
+	sort.Strings(appNames)
 	apps := make([]appJSON, 0, len(localSnap.Apps))
-	for _, app := range localSnap.Apps {
+	for _, appName := range appNames {
+		app := localSnap.Apps[appName]
```

Comment:
it's probably easier and clearer to leave this bit as it was, and then add a sort.Slice call at the end